### PR TITLE
chore: split out CI into unit tests + extra checks

### PR DESCRIPTION
**Pull request by jackfranklin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ services: xvfb
 
 jobs:
   include:
-
     - os: "osx"
+      name: 'Unit tests: OSX/Chromium'
```
Comment (mathiasbynens) [inline .travis.yml]:
```suggestion
      name: 'Unit tests: macOS/Chromium'
```

**Review by mathiasbynens — APPROVED** [review]
Nice!

File: .travis.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ jobs:
         - npm run unit
 
     - os: "windows"
+      name: 'Unit tests: Windows/Chromium'
+      node_js: "10.19.0"
```
Comment (mathiasbynens) [inline .travis.yml:20]: This seems duplicated:

```suggestion
```

Comment (jackfranklin) [reply .travis.yml:20]: Gotta put it twice to be _extra sure_ on the version 🤦

File: .travis.yml [hunk 3]
```diff
@@ -26,15 +27,31 @@ jobs:
...
       script:
         - npm run test-install
-        - travis/chromium.sh
+        - npm run coverage
+
+    # This bot runs all the extra checks that aren't the main Puppeteer unit tests
+    - node_js: "10.19.0"
+      name: 'Extra tests: Linux/Chromium'
+      env:
+        -CHROMIUM=true
```
Comment (mathiasbynens) [inline .travis.yml]:
```suggestion
        - CHROMIUM=true
```

**Pull request merged**